internal/govulncheckshim/vulncheckshim.go

```diff
@@ -0,0 +1,76 @@
+package govulncheckshim
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/osv-scanner/pkg/models"
+	"golang.org/x/tools/go/packages"
+	"golang.org/x/vuln/exp/govulncheck"
+	"golang.org/x/vuln/vulncheck"
+)
+
+type packageError struct {
+	Errors []packages.Error
+}
+
+func (e *packageError) Error() string {
+	var b strings.Builder
+	fmt.Fprintln(&b, "Packages contain errors:")
+	for _, e := range e.Errors {
+		fmt.Fprintln(&b, e)
+	}
+
+	return b.String()
+}
+
+// RunGoVulnCheck runs govulncheck with a subset of vulnerabilities identified by osv-scanner
+func RunGoVulnCheck(path string, vulns []models.Vulnerability) (*govulncheck.Result, error) {
```

Comment:
Are there any basic tests we can add for this?

Reply:
bumping this comment :)

Reply:
Added a test for a small example project